bin/run-tests.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+#
+# Run all tests including integration tests via docker.
+#
+# The script places some files into target/docker-build-volumes.
+# If you keep it around between builds your can greatly speed up resolving artifacts
+# but on the other hand you do not test if all artifacts are still resolvable.
+#
+
+# be verbose and print all commands, better for debugging
+set -x
+
+PROJECT_DIR=`dirname $0`/..
+PROJECT_DIR=$(cd "$PROJECT_DIR"; pwd)
+
+if [ -z "$1" ]; then
+    echo "Usage:" >&2
+    echo "$0 <unique build id without crazy characters or spaces>" >&2
```

Comment:
Would it make sense to create a buildId, if it is omitted?
As I started it the first time, my question was: where do I get the buildId....